packages/@okta/vuepress-site/docs/guides/federated-claims/main/index.md

```diff
@@ -0,0 +1,171 @@
+
+Other attributes included with an entitlement are the value or values that it may contain. These are the specific values that the principal (user) has for each entitlement. Values within an entitlement may be either a single string or an array of strings. These values have two important attributes, similar to the entitlement itself:
+
+* `name`: The display name of the entitlement value
+* `externalValue`: External system representation of the value
+
+### Expression Language and entitlements
+
+The integration of entitlements into [Expression Language in Identity Engine](/docs/reference/okta-expression-language-in-identity-engine/) is on the `appuser` context. When you [configure an entitlement claim for an app](#configure-an-entitlement-claim-for-the-app), the expression states which entitlement the SP should evaluate for the principal and the app.
+
+If the `externalValue` of an entitlement is `permission`, then your EL expression would be `appuser.entitlement.permission`, because `entitlement.externalValue = permission`. At evaluation time, the results of the expression are the entitlement values that the principal has been granted for the entitlement referenced in the expression.
+
+The following is an example JSON body of a POST request to `https://{yourOktaDomain}/api/v1/apps/{appID}/federated-claims` to create an entitlement:
+
+```JSON
+  {
+     "name": "entitlement_claim_name",
+     "expression": "appuser.entitlements.example_entitlement"
+  }
+```
+
+## Configure entitlements
+
+The following sections are an example flow for setting up and using entitlements for your SAML or OIDC app.
+
+## Update your app to support entitlements
+
+> **Note:** Identity Governance doesn't support **Federated Broker Mode** for OIDC apps.
+
+1. Go to **Applications** > **Applications** and select the app that you want to define entitlements for.
+1. On the **General** tab, scroll down to the **Identity Governance** section and click **Edit**.
+1. Select **Enabled** to enable the Governance Engine, and then click **Save**. The **Governance** tab should appear within a few seconds. If it doesn't, refresh the page.
+
+## Define entitlements for the app
+
+> **Note:** Entitlements that you create here are what you use in the [EL expressions](/docs/reference/okta-expression-language-in-identity-engine/) that then insert the claims into the attribute statement or ID token.
+
+1. While still in the app that you updated in the previous section, select the **Governance** tab.
+1. On the **Entitlements** tab, click **Add Entitlement** and define the entitlement properties:
+
+   * Enter a **Display name** for the entitlement. In this example flow, enter **Permission**.
+   * Enter a **Variable name** for the entitlement. In this example flow, enter **permission**. Use this attribute in your EL expression when you [configure an entitlement claim for the app](#configure-an-entitlement-claim-for-the-app).
```

Comment:
Double-checking that line 84 should use "Permission" and 85 should use "permission"?